Linux/Car_Client/chronoscomm.cpp

```diff
@@ -617,8 +658,8 @@ void ChronosComm::readPendingDatagrams()
 
         vb.remove(vb.size() - 2, 2);
 
-        if (true || checksum == 0) {
-            decodeMsg(message_id, message_len, vb, sender_id);
+        if (true /* || checksum == 0 */) {
```

Comment:
Is this temporary, or could we remove the if-statement?

Reply:
It's mostly as a reminder to check the checksum when that calculation has been implemented.